sushi-config.yaml

```diff
@@ -34,6 +34,9 @@ publisher:
 dependencies:
   fhir.org.nz.ig.base: 2.0.0
   hl7.fhir.uv.sdc: 3.0.0
+  # hl7.org.nz.fhir.ig.hpi:
```

Comment:
This doesn't work right? Should probably remove the commented out section to remove confusion

Reply:
It sure doesn't work!  Removed